fs-storage/src/utils.rs

```diff
@@ -51,6 +53,56 @@ where
     Ok(data)
 }
 
+/// Writes a serializable value to a file and returns the timestamp of the write
+pub fn write_json_file<T: Serialize>(
+    path: &Path,
+    value: &T,
+    time: SystemTime,
+) -> Result<()> {
```

Comment:
I think accepting a timestamp in this function could lead to confusion. I checked all its usages, and it seems we’re always using `Time::now()` to get the timestamp. 

I think it would be better to remove the timestamp parameter and just call `Time::now()` directly inside the function.

Reply:
Tried doing it, but it failed some tests due to sync issues in both `file_storage` and `folder_storage`.